### DISCUSSION TITLE
API for changing Org settings

### DISCUSSION THREAD

**Discussion by sankbab** [community/community#61219] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

Hi Team,

We are in process of changing the Org settings for our org and want to change some buld changes which would require to do via an API REST call. Is there a way to do it?  For eg: we need to limi the "Runners" under "Code Planning and automation" -> Actions -> General -> to Selected Repositories instead of All repositories. We have a list of repositories and need to change it via API, because manual editting and selecting is very much time taken process. Please help.

Thank you.

**Comment by byrneh** [comment]
@sankbab there are a couple of REST API endpoints related to Action available for doing this.

[#set-github-actions-permissions-for-an-organization](https://docs.github.com/en/rest/actions/permissions?apiVersion=2022-11-28#set-github-actions-permissions-for-an-organization)
Sets the GitHub Actions permissions policy for repositories and allowed actions and reusable workflows in an organization.

[#set-selected-repositories-enabled-for-github-actions-in-an-organization](https://docs.github.com/en/rest/actions/permissions?apiVersion=2022-11-28#set-selected-repositories-enabled-for-github-actions-in-an-organization)
Replaces the list of selected repositories that are enabled for GitHub Actions in an organization.

**Comment by sankbab** [comment]
Thank you @byrneh. It worked.
